1.max_project_images/README.md

```diff
@@ -0,0 +1,13 @@
+# Maximum Projection of z-Stack Images
+
+## Description
+This script is used to generate maximum projection images from z-stack images. The script will generate a new folder named "max_project_images" in the same directory as the input images. The maximum projection images will be saved in the new folder.
```

Comment:
```suggestion
This script is used to generate maximum projection images from z-stack images. 
The script will generate a new folder named "max_project_images" in the same directory as the input images. 
The maximum projection images will be saved in the new folder.
```
Recommend to make sure that each sentence is a new line.

Reply:
Also, I would recommend including what software you are using to perform maximum projection in this README explicitly (unless I missed it).